src/main/java/seedu/address/model/task/Name.java

```diff
@@ -10,14 +10,14 @@
 public class Name implements Comparable<Name> {
 
     public static final String MESSAGE_CONSTRAINTS =
-            "Names should only contain alphanumeric characters and spaces, and it should not be blank";
+            "Names should only contain alphanumeric characters and spaces, and it should contain 1-255 characters.";
```

Comment:
Will it be better to say "Name should only contain alphanumeric characters and spaces, and it has a character limit of 1-255 characters."?